tools/test/travis-ci/scancode_evaluate_test.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# Copyright (c) 2020 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+import importlib
+import os
+import pytest
+
+# TODO: fix scancode to match python naming conventROOTi
+license_check = importlib.import_module("scancode-evaluate").license_check
+
+ROOT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__))
+)
+
+# path to stub files
+stub_path = ROOT + "/scancode_test/"
+
+# template copyright notices
+header_without_spdx = "/* Copyright (C) Arm Limited, Inc - All Rights Reserved\
+ * Unauthorized copying of this. file, via any medium is strictly prohibited\
+ * Proprietary and confidential\
+ */"
+
+header_with_spdx = "/* mbed Microcontroller Library\
+ * Copyright (c) 2006-2013 ARM Limited\
+ *\
+ * SPDX-License-Identifier: Apache-2.0\
+ * Licensed under the Apache License, Version 2.0 (the \"License\");\
+ * you may not use this file except in compliance with the License.\
+ * You may obtain a copy of the License at\
+ *\
+ *     http://www.apache.org/licenses/LICENSE-2.0\
+ *\
+ * Unless required by applicable law or agreed to in writing, software\
+ * distributed under the License is distributed on an \"AS IS\" BASIS,\
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\
+ * See the License for the specific language governing permissions and\
+ * limitations under the License.\
+ */"
+
+@pytest.fixture()
+def resource_test3():
+    """Create stub files.
+    test3.h missing license notice
+    test4.h with license notice
+    test5.h with license notice
+    """
+    file_paths = [
+        stub_path + "test3.h",
+        stub_path + "test4.h",
+        stub_path + "test5.h",
+    ]
+    for file_path in file_paths:
+        with open(file_path, "w") as new_file:
+            if file_path in [stub_path + "test4.h", stub_path + "test5.h"]:
+                new_file.write(header_with_spdx)
+            else:
+                new_file.write(header_without_spdx)
+    yield "resource_test3"
+    # delete stub files
+    for file_path in file_paths:
+        os.remove(file_path)
+
+
+@pytest.fixture()
+def resource_test4():
+    """Create stub files.
+    test.h missing license notice
+    """
+    file_paths = [
+        stub_path + "test.h",
+    ]
+    for file_path in file_paths:
+        with open(file_path, "w") as new_file:
+                new_file.write(header_without_spdx)
+    yield "resource_test4"
+    # delete stub files
+    for file_path in file_paths:
+        os.remove(file_path)
+
+
+class TestScancodeEvaluate:
+    """ Test scancode evaluation script """
+    def test_scancode_case_1(self):
```

Comment:
Can we name the test cases with names that describe what they're actually doing? Makes it much easier to see what failed in the test runner output.